### ISSUE TITLE
Cannot run website with serve-docs.sh

### ISSUE THREAD

**Issue by steren** [bazelbuild/bazel-website#28]
I get this error:

````
steren:bazel-website steren$ ./scripts/serve-docs.sh
expr: syntax error
INFO: Found 1 target...
ERROR: /Users/steren/work/bazel-website/BUILD:84:1: Generating site tarball. failed: Process exited with status 1 [sandboxed].
Cannot create redirect file docs/skylark/lib/provider.html. File exists.
Use --strategy=SiteTar=standalone to disable sandboxing for the failing actions.
Target //:jekyll-tree.tar failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 3.448s, Critical Path: 1.76s
````

**Comment by davidzchen** [comment]
+cc @laurentlb, @fweikert 

Are you running on macOS? The problem is that when we generate the Skylark Library, there is both a [`provider.html`](https://bazel.build/versions/master/docs/skylark/lib/provider.html) and a [`Provider.html`](https://bazel.build/versions/master/docs/skylark/lib/Provider.html). Because HFS+ (and presumably the upcoming APFS) is by default case-insensitive, we get this error.

The script runs without any issues on Linux. IMO the ideal fix for this is to either improve the directory structure of the Skylark Library to separate modules and types or have a single page for both the `provider` module and the `Provider` type.

**Comment by steren** [comment]
Yes I am on macOS.

It is a bad practice to have 2 web page URLs that only differ by a letters being upper case.

**Comment by davidzchen** [comment]
Agreed. Let's open a bug on bazelbuild/bazel to fix this in the Skylark Library.

**Comment by jin** [comment]
This is unfortunately still an issue. For local development, I'd just comment out either page from redirects.bzl.
